Version 0.14.31 does not compile on FreeBSD anymore

Was there a change from version 0.14.29 to 0.14.31 that can explain why the build fails now on FreeBSD?

I see errors like:
```
In file included from ./common.h:31:
/usr/local/include/ev++.h:555:15: warning: mangled name of 'set_allocator' will change in C++17 due to non-throwing exception specification in function signature [-Wc++17-compat-mangling]
  inline void set_allocator (void *(*cb)(void *ptr, long size) throw ()) throw ()
              ^
/usr/local/include/ev++.h:560:15: warning: mangled name of 'set_syserr_cb' will change in C++17 due to non-throwing exception specification in function signature [-Wc++17-compat-mangling]
  inline void set_syserr_cb (void (*cb)(const char *msg) throw ()) throw ()
              ^
cm_ip.cpp:46:25: error: member access into incomplete type 'const struct sockaddr_in'
      buf[2] = (ntohl (a->sin_addr.s_addr) >> 24) & 0xff;
                        ^
./cm_ip.h:33:35: note: forward declaration of 'sockaddr_in'
CArray IPtoEIBNetIP (const struct sockaddr_in *a, bool nat);
                                  ^
cm_ip.cpp:47:25: error: member access into incomplete type 'const struct sockaddr_in'
      buf[3] = (ntohl (a->sin_addr.s_addr) >> 16) & 0xff;
                        ^
./cm_ip.h:33:35: note: forward declaration of 'sockaddr_in'
CArray IPtoEIBNetIP (const struct sockaddr_in *a, bool nat);
                                  ^
cm_ip.cpp:48:25: error: member access into incomplete type 'const struct sockaddr_in'
      buf[4] = (ntohl (a->sin_addr.s_addr) >> 8) & 0xff;
                        ^
./cm_ip.h:33:35: note: forward declaration of 'sockaddr_in'
CArray IPtoEIBNetIP (const struct sockaddr_in *a, bool nat);
                                  ^
cm_ip.cpp:49:25: error: member access into incomplete type 'const struct sockaddr_in'
      buf[5] = (ntohl (a->sin_addr.s_addr) >> 0) & 0xff;
                        ^
./cm_ip.h:33:35: note: forward declaration of 'sockaddr_in'
CArray IPtoEIBNetIP (const struct sockaddr_in *a, bool nat);
                                  ^
cm_ip.cpp:50:25: error: member access into incomplete type 'const struct sockaddr_in'
      buf[6] = (ntohs (a->sin_port) >> 8) & 0xff;
                        ^
./cm_ip.h:33:35: note: forward declaration of 'sockaddr_in'
CArray IPtoEIBNetIP (const struct sockaddr_in *a, bool nat);
                                  ^
cm_ip.cpp:51:25: error: member access into incomplete type 'const struct sockaddr_in'
      buf[7] = (ntohs (a->sin_port) >> 0) & 0xff;
                        ^
./cm_ip.h:33:35: note: forward declaration of 'sockaddr_in'
CArray IPtoEIBNetIP (const struct sockaddr_in *a, bool nat);
                                  ^
cm_ip.cpp:61:24: error: invalid application of 'sizeof' to an incomplete type 'struct sockaddr_in'
  memset (a, 0, sizeof (*a));
                       ^~~~
./cm_ip.h:33:35: note: forward declaration of 'sockaddr_in'
CArray IPtoEIBNetIP (const struct sockaddr_in *a, bool nat);
                                  ^
cm_ip.cpp:67:4: error: member access into incomplete type 'struct sockaddr_in'
  a->sin_len = sizeof (*a);
   ^
./cm_ip.h:33:35: note: forward declaration of 'sockaddr_in'
CArray IPtoEIBNetIP (const struct sockaddr_in *a, bool nat);
                                  ^
cm_ip.cpp:69:4: error: member access into incomplete type 'struct sockaddr_in'
  a->sin_family = AF_INET;
   ^
./cm_ip.h:33:35: note: forward declaration of 'sockaddr_in'
CArray IPtoEIBNetIP (const struct sockaddr_in *a, bool nat);
                                  ^
cm_ip.cpp:71:6: error: member access into incomplete type 'struct sockaddr_in'
    a->sin_port = src->sin_port;
     ^
./cm_ip.h:33:35: note: forward declaration of 'sockaddr_in'
CArray IPtoEIBNetIP (const struct sockaddr_in *a, bool nat);
                                  ^
cm_ip.cpp:73:6: error: member access into incomplete type 'struct sockaddr_in'
    a->sin_port = htons (port);
     ^
./cm_ip.h:33:35: note: forward declaration of 'sockaddr_in'
CArray IPtoEIBNetIP (const struct sockaddr_in *a, bool nat);
                                  ^
cm_ip.cpp:77:8: error: member access into incomplete type 'struct sockaddr_in'
      a->sin_addr.s_addr = src->sin_addr.s_addr;
       ^
./cm_ip.h:33:35: note: forward declaration of 'sockaddr_in'
CArray IPtoEIBNetIP (const struct sockaddr_in *a, bool nat);
                                  ^
cm_ip.cpp:80:6: error: member access into incomplete type 'struct sockaddr_in'
    a->sin_addr.s_addr = htonl (ip);
     ^
./cm_ip.h:33:35: note: forward declaration of 'sockaddr_in'
CArray IPtoEIBNetIP (const struct sockaddr_in *a, bool nat);
                                  ^
6 warnings and 13 errors generated.
```
The full build log is here:
https://pkg.fechner.net/data/121amd64-gitlab/2020-02-23_11h51m02s/logs/errors/knxd-0.14.31.log

Anyone can help please?

--- a/AUTHORS
+++ b/AUTHORS
@@ -41,3 +41,4 @@ Othmar Truniger <github@truniger.ch>
 Matthias Urlichs <matthias@urlichs.de>
 Christian Wicke <knx.christian@safersignup.com>
 Timo Wingender <github@timo-wingender.de>
+Matthias Fechner <idefix@fechner.net>

--- a/src/libserver/cm_ip.cpp
+++ b/src/libserver/cm_ip.cpp
@@ -21,6 +21,7 @@
 #include "config.h"
 
 #include <net/if.h>
+#include <netinet/in.h>
 #include <netdb.h>
 #include <sys/socket.h>
 #include <unistd.h>

